db/migrate/20230403013014_add_default_publicity.rb

```diff
@@ -0,0 +1,9 @@
+class AddDefaultPublicity < ActiveRecord::Migration[7.0]
+  def up
+    change_column :rooms, :publicity_level, :string, default: :listed, null: false
```

Comment:
I'm trying to decide whether `:listed` or `:unlisted` is the better default, but I am landing on 🤷

Reply:
Yeah, I wasn't sure either, but if feels like a `public` default sort of implies a `listed` default?

Reply:
That makes sense to me!